zproject/urls.py

```diff
@@ -573,14 +574,16 @@
     path('integrations/doc-html/<integration_name>',
          zerver.views.documentation.integration_doc,
          name="zerver.views.documentation.integration_doc"),
-    re_path(r'^integrations/(.*)$', IntegrationView.as_view()),
+    path('integrations/', integrations),
+    path('integrations/<path:path>', integrations),
 
     # Landing page, features pages, signup form, etc.
     path('hello/', zerver.views.portico.hello_view, name='landing-page'),
     path('new-user/', RedirectView.as_view(url='/hello', permanent=True)),
     path('features/', zerver.views.portico.landing_view, {'template_name': 'zerver/features.html'}),
     path('plans/', zerver.views.portico.plans_view, name='plans'),
-    re_path(r'apps/(.*)$', zerver.views.portico.apps_view, name='zerver.views.home.apps_view'),
+    path('apps/', zerver.views.portico.apps_view, name='zerver.views.home.apps_view'),
+    path('apps/<path:path>', zerver.views.portico.apps_view, name='zerver.views.home.apps_view'),
```

Comment:
Do we want this to just be `apps/<platform>`?  I don't think we have a use case for longer paths there.

Reply:
Changed.